<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditSummonerName"/>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBoxCampeones"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>IDS Campeones</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonGetCampeonByID">
      <property name="text">
       <string>Buscar Campeon por ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButtonSearchSummoner">
      <property name="text">
       <string>Buscar Invocador</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditIdCampeon"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBoxServer"/>
    </item>
    <item row="5" column="2">
     <widget class="QTextEdit" name="textEditResult"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Servidor</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelSumonerImage">
      <property name="text">
       <string>Imagen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCampeones">
    <property name="title">
     <string>Campeones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInvocadores">
    <property name="title">
     <string>Invocadores</string>
    </property>
   </widget>
   <addaction name="menuCampeones"/>
   <addaction name="menuInvocadores"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
